lib/membrane/rtp/vad_utils/is_speaking_estimator.ex

```diff
@@ -0,0 +1,118 @@
+defmodule Membrane.RTP.VadUtils.IsSpeakingEstimator do
+  @moduledoc """
+  Module for estimating if the user is speaking based on
+  "Dominant Speaker Identification for Multipoint Videoconferencing
+  by Ilana Volfin and Israel Cohen"
+  """
+
+  # number of levels inside one immediate interval
+  @n1 1
+  # number of immediate intervals inside one medium interval
+  @n2 10
+  # number of medium intervals inside one long interval
+  @n3 5
+
+  @immediate_score_threshold 0
+  @medium_score_threshold 20
+  @long_score_threshold 20
+
+  @min_levels_length @n1 * @n2 * @n3
+
+  # @max_level 127
+  # @min_level 0
+  @medium_threshold 1
+  @long_threshold 3
+
+  @min_activity_score 1.0e-8
+
+  defp binomial_coefficient(n, k) when k < 0 or k > n, do: 0
+  defp binomial_coefficient(n, k) when k == 0 or n == k, do: 1
+
+  defp binomial_coefficient(n, k) do
+    k = min(k, n - k)
+
+    Enum.to_list(1..k)
+    |> Enum.reduce(1, fn i, acc -> div(acc * (n - i + 1), i) end)
```

Comment:
More about the binomial coefficient:
   - it is based on the implementation given on wikipedia [here](https://en.wikipedia.org/wiki/Binomial_coefficient#In_programming_languages) (the second code snippet with iterative implementation in python)
   - as noted in the conversation with @bblaszkow06, the values evaluated here are no bigger than `max(@n1, @n2, @n3)`. This means this could be evaluated in compile time.